Multisig/Data/Services/SafeTransactionService.swift

```diff
@@ -55,7 +55,11 @@ class SafeTransactionService {
         return try httpClient.execute(request: SafeBalancesRequest(address: address))
     }
 
-    func transactions(address: Address, offset: Int = 0, limit: Int = 100) throws -> TransactionsRequest.Response {
+    func transactions(address: Address?, offset: Int? = 0, limit: Int? = 20, url: String? = nil) throws -> TransactionsRequest.Response {
```

Comment:
I'd create 2 different methods instead - one with all parameters, one with just the URL, because they're mutually exclusive